app/services/DomainService.scala

```diff
@@ -16,7 +16,7 @@
 
 package services
 
-import domain.{Declaration, GuaranteeTransaction, CashTransactions, CashDailyStatement, Amounts, Transaction, TaxGroup}
+import domain.{Amounts, CashDailyStatement, CashTransactions, Declaration, GuaranteeTransaction, TaxGroup, TaxTypeHolder, Transaction}
```

Comment:
I feel like we're using loads of imports here - is it easier just to say import domain.*